diff --git a/core/themes/stable/templates/admin/status-report.html.twig b/core/themes/stable/templates/admin/status-report.html.twig
index 7f4c600..3e5b28e 100644
--- a/core/themes/stable/templates/admin/status-report.html.twig
+++ b/core/themes/stable/templates/admin/status-report.html.twig
@@ -15,25 +15,61 @@
  * @see template_preprocess_status_report()
  */
 #}
-<table class="system-status-report">
-  <tbody>
-  {% for requirement in requirements %}
-    <tr class="system-status-report__entry system-status-report__entry--{{ requirement.severity_status }} color-{{ requirement.severity_status }}">
+ <table class="system-status-report">
+  <tbody> 
+    <span> General System Information </span>    
+      {% for key,requirement in requirements %}
+        {% if key == 0 or key == 3 or key == 5 or key == 14 or key == 24 %}
+          <tr>
+              <td class="system-status-report__status-title">
+                {{ requirement.title }}
+              </td>
+              <td>
+                {{ requirement.value }}
+                {% if requirement.description %}
+                  <div class="description">{{ requirement.description }}</div>
+                {% endif %}
+              </td>
+            </tr>
+        {% endif %}
+      {% endfor %} 
+    {# print error  #}
+    {% for requirement in requirements %}
+    
       {% if requirement.severity_status in ['warning', 'error'] %}
-        <th class="system-status-report__status-title system-status-report__status-icon system-status-report__status-icon--{{ requirement.severity_status }}">
+        <tr class="system-status-report__entry system-status-report__entry--{{ requirement.severity_status }} color-{{ requirement.severity_status }}">
+        <td class="system-status-report__status-title system-status-report__status-icon system-status-report__status-icon--{{ requirement.severity_status }}">
           <span class="visually-hidden">{{ requirement.severity_title }}</span>
-      {% else %}
-        <th class="system-status-report__status-title">
-      {% endif %}
-        {{ requirement.title }}
-      </th>
-      <td>
+          <td class="system-status-report__status-title">
+            {{ requirement.title }}
+          </td>  
+        <td>
         {{ requirement.value }}
         {% if requirement.description %}
           <div class="description">{{ requirement.description }}</div>
         {% endif %}
-      </td>
-    </tr>
-  {% endfor %}
+         </td> 
+       </td>
+     </tr>            
+       {% endif %} 
+     
+   {% endfor %}
+
+    {% for key,requirement in requirements %}
+        {% if key != 0 and key != 3 and key != 5 and key != 14 and key != 24 %}
+          <tr>
+            <td class="system-status-report__status-title">
+                {{ requirement.title }}
+              </td>
+              <td>
+                {{ requirement.value }}
+                {% if requirement.description %}
+                  <div class="description">{{ requirement.description }}</div>
+                {% endif %}
+            </td> 
+          </tr>
+        {% endif %}
+    {% endfor %} 
   </tbody>
 </table>
+ 
\ No newline at end of file
